tests/engines/registries/acceptance/branded/moderation/moderators-test.ts

```diff
@@ -63,4 +68,129 @@ module('Registries | Acceptance | branded.moderation | moderators', hooks => {
+                'registries.moderation.moderators.updatedModeratorPermissionSuccess',
+                { userName: suzy.fullName, permission: 'admin' },
+            ),
+            'Toast successful message for updating permission',
+        );
+        assert.dom(`[data-test-moderator-row="${suzy.id}"]>div>[data-test-permission-group]`).hasText(
+            'Admin',
+            'Suzy is now an admin',
+        );
+        // Removing Suzy as a moderator
+        await click(`[data-test-delete-moderator-button="${suzy.id}"]>[data-test-delete-button]`);
+        await click('[data-test-confirm-delete]');
+        assert.dom('#toast-container', document as any).hasTextContaining(
+            t(
+                'registries.moderation.moderators.removedModeratorSuccess',
+                { userName: suzy.fullName },
+            ),
+            'Toast successful message for removing moderator',
+        );
+        assert.dom('[data-test-moderator-row]').exists({ count: 2 }, 'There are two moderators');
+        assert.dom(`[data-test-moderator-row="${suzy.id}"]`).doesNotExist('Suzy is no longer a moderator');
+    });
+
+    test('moderator CRUD operations failure should toast', async assert => {
+        server.namespace = '/v2';
+        server.post('/providers/registrations/:parentID/moderators', () => ({
+            errors: [{ detail: '' }],
+        }), 400);
+        server.put('/providers/registrations/:parentID/moderators/:id/', () => ({
+            errors: [{ detail: '' }],
+        }), 400);
+        server.patch('/providers/registrations/:parentID/moderators/:id/', () => ({
+            errors: [{ detail: '' }],
+        }), 400);
+        server.del('/providers/registrations/:parentID/moderators/:id/', () => ({
+            errors: [{ detail: '' }],
+        }), 400);
+        const regProvider = server.schema.registrationProviders.find('empty');
+        const currentUser = server.create('user', { fullName: 'J' }, 'loggedIn');
+        server.create('user', { fullName: 'Bae Suzy' });
+        server.create('moderator', { id: currentUser.id, user: currentUser, provider: regProvider }, 'asAdmin');
+        regProvider.update({ permissions: ['view_submissions'] });
+        await visit('/registries/empty/moderation/moderators');
+        // add failure should toast
+        await click('[data-test-add-moderator-button]');
+        await selectSearch('[data-test-select-user]', 'Suzy');
+        await selectChoose('[data-test-select-user]', 'Bae Suzy');
+        await selectChoose('[data-test-select-permission]', 'Moderator');
+        await click('[data-test-confirm-add-moderator-button]');
+        assert.dom('#toast-container', document as any).hasTextContaining(
+            t(
+                'registries.moderation.moderators.addedNewModeratorError',
+                { permission: 'moderator' },
+            ),
+            'Toast error message for adding user as moderator failure',
+        );
+        await timeout(5000);
```

Comment:
Do we need these timeouts? Are error toasts and success toasts different in some way?